### PR TITLE
Small improvements to GVRPickedObject.

### PR DESCRIPTION
**Pull request by johnspurlock**
- Make constructor visible, take struct fields as direct params.
 - Use standard naming for private fields.
 - Fix some javadocs.

GearVRf-DCO-1.0-Signed-off-by: John Spurlock
john.spurlock@gmail.com

### CODE REVIEW

**Comment by liaxim** [comment]
Looks good

**Comment by johnspurlock** [comment]
Thanks - can you merge?  I don't have write access.

File: GVRf/Framework/src/org/gearvrf/GVRPicker.java [hunk 1]
```diff
@@ -336,12 +336,12 @@ public static final float pickSceneObject(GVRSceneObject sceneObject,
...
     public static class GVRPickedObject {
-        private final GVRSceneObject sceneObject;
-        private final float[] hitLocation;
+        private final GVRSceneObject mHitObject;
+        private final float[] mHitLocation;
 
-        private GVRPickedObject(GVREyePointeeHolder holder) {
-            sceneObject = holder.getOwnerObject();
-            hitLocation = holder.getHit();
+        public GVRPickedObject(GVRSceneObject hitObject, float[] hitLocation) {
```
Comment (thomasflynn) [inline GVRf/Framework/src/org/gearvrf/GVRPicker.java]: this went from private to public.  i'm assuming there's a good reason?  and if so, all public methods of all public classes need javadoc.

File: GVRf/Framework/src/org/gearvrf/GVRPicker.java [hunk 2]
```diff
@@ -362,22 +362,22 @@ public GVRSceneObject getHitObject() {
...
         }
 
         /** The x coordinate of the hit location */
         public float getHitX() {
-            return hitLocation[0];
+            return mHitLocation[0];
         }
 
-        /** The x coordinate of the hit location */
+        /** The y coordinate of the hit location */
```
Comment (thomasflynn) [inline GVRf/Framework/src/org/gearvrf/GVRPicker.java]: haha, good catch.  not sure how we missed these two copy-n-paste typos.  thanks.

**Comment by thomasflynn** [comment]
needs javadoc.  otherwise, looks ok.  and only reviewers/maintainers can merge code:
http://wiki.gearvrf.org/bin/view/GearVRF/GearVRfWikiCommunity#ProjectGovernance

**Comment by johnspurlock** [comment]
Ok, added javadoc for the ctor, please take another look.

Use case was modifying the list of hit objects returned during frame processing, removing some, adding some synthetic ones for related objects.   Noticed that this is basically an immutable struct / pair class, and it would be useful to reuse instead of creating another class.  Might also be useful for unit testing, but that's definitely not what I was doing.  : )

**Comment by thomasflynn** [comment]
ok, fair enough.  looks good now.  and feel free to write unit tests if that's what you enjoy doing ;-)

**Pull request merged**